odh-jupyter-trash-cleanup/ui-tests/playwright.config.js

```diff
@@ -2,13 +2,31 @@
  * Configuration for Playwright using default from @jupyterlab/galata
  */
 const baseConfig = require('@jupyterlab/galata/lib/playwright-config');
+const fs = require('fs');
+const path = require('path');
+const GALATA_ROOT = path.resolve(__dirname, '..', '..', '.galata-root');
+
+try {
+  fs.rmSync(GALATA_ROOT, { recursive: true, force: true });
+} catch {}
```

Comment:
empty catch - let's print some error log or something to communicate something went wrong